tendermint/src/rpc/event_listener.rs

```diff
@@ -191,7 +192,8 @@ pub struct Attribute {
 ///Block Value
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct EventDataNewBlock {
-    block: Option<Block>,
+    /// block
```

Comment:
```suggestion
    /// The new block
```

Reply:
Yeah, this change wasn't meant to be there, sorry